driver-core/src/main/java/com/datastax/driver/core/querybuilder/Delete.java

```diff
@@ -466,6 +495,36 @@ boolean containsBindMarker() {
 
     }
 
+    private static class ListElementSelector extends CollectionElementSelector {
+
+        ListElementSelector(String columnName, Object key) {
+            super(columnName, key);
+        }
+    }
+
+    private static class SetElementSelector extends CollectionElementSelector {
```

Comment:
Set and Map `ElementSelector` are not useful now but it looked cleaner to add them imo and they could be useful for the future.